docker/Dockerfile.build

```diff
@@ -1,25 +1,7 @@
-FROM debian:jessie
+FROM debian:stretch
```

Comment:
Great  - no problem, but I wonder if this is a push to get our other TrueNTH systems up to stretch to be in alignment?

Reply:
It would be nice, but luckily it looks like there isn't much of a difference to our portal code. This did serve well as a smoke-test though :)